static/app/views/replays/detail/network/index.tsx

```diff
@@ -172,7 +205,7 @@ function NetworkList() {
           }}
         >
           {networkFrames ? (
-            <OverflowHidden>
+            <OverflowHidden style={{display: 'grid'}}>
```

Comment:
Can you put this css property down inside the definition of `OverflowHidden`? Looks like `<OverflowHidden>` is only used once, so we don't have to worry about unintended side-effect.

```
const OverflowHidden = styled('div')`
  display: grid;
  position: relative;
  height: 100%;
  overflow: hidden;
`;
```